operatorcourier/api.py

```diff
@@ -41,42 +41,29 @@ def build_and_verify(source_dir=None, yamls=None, ui_validate_io=False,
     :raises OpCourierBadBundle: When the resulting bundle fails validation
     """
 
-    if source_dir is not None and yamls is not None:
-        logger.error("Both source_dir and yamls cannot be defined.")
-        raise TypeError(
-            "Both source_dir and yamls cannot be specified on function call.")
+    manifests = get_manifests_info(source_dir, yamls)
+    validation_dict, bundle_dict \
+        = get_validation_dict_from_manifests(manifests, ui_validate_io, repository)
```

Comment:
function is named `get_validation_dict_*` but returns `bundle_dict` too. It is quite messy IMHO

Reply:
@MartinBasti Thank you for the comment! I have updated and set `bundle_dict` as an instance variable instead.